error/s2n_errno.c

```diff
@@ -370,10 +370,22 @@ int s2n_stack_traces_enabled_set(bool newval)
 #ifdef S2N_HAVE_EXECINFO
 
 #define MAX_BACKTRACE_DEPTH 20
-__thread struct s2n_stacktrace tl_stacktrace = {0};
+__thread struct s2n_stacktrace tl_stacktrace;
+__thread bool s2n_stacktrace_initialized;
+
+int s2n_init_stacktrace(void)
```

Comment:
Could you initialize in [s2n_init](https://github.com/aws/s2n-tls/blob/main/utils/s2n_init.c) instead?

Reply:
Have you been able to run this through gdb? That might provide some insights into what is going on with s2n_init.